packages/site/src/layout/SearchBox.tsx

```diff
@@ -41,8 +41,10 @@ export const SearchBox = ({
   const filteredComponentList = useMemo(() => {
     return componentList.filter(
       d =>
-        d.title.includes(search) ||
-        d.additionalMatches?.find(e => e.includes(search)),
+        d.title.toLowerCase().includes(search.toLowerCase()) ||
```

Comment:
Awesome!! Dude if you're fixing bugs in the site I take it as a good sign the code is understandable. :)